earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/geecheck_tests/user_tests/disk_space_test.py

```diff
@@ -40,9 +40,9 @@ def getDiskInfo():
   mount_point = getMountPoint(asset_root)
 
   available_space, size = getFsFreespace(mount_point)
-  percentage_used = (size - available_space) * 100 / size
+  percentage_avail = 100 - ((size - available_space) * 100 / size)
```

Comment:
this function was computing used space and not free space ?

Reply:
if we change this function to return available space, we need to update its description in line 31. Or we can keep this function as it is and update `TestDiskSpace` function to verify the expected percent used (80% or less) ?

Reply:
`getDiskInfo` seems a little generic, we should probably update to reflect what the function does. ex. `getAvailableDiskInfo` or `getUsedDiskInfo`.

Reply:
Changed getDiskInfo to getDiskFreeSpace (since it returns the % of free space available) and getFsFreeSpace to getFsInfo (since it returns two values, the total size and available space)